get-pmm.sh

```diff
@@ -223,19 +223,25 @@ run_docker() {
 # If any PMM server instance is run - stop and backup it.
 #######################################
 start_pmm() {
-  msg "Starting PMM server..."
+  msg "Fetching docker image...\n"
   run_docker "pull $repo:$tag 1> /dev/null"
 
+  msg "Checking data volume...\n"
+  pmm_archive="$(date "+%F-%H%M%S")"
   if ! run_docker "inspect pmm-data 1> /dev/null 2> /dev/null"; then
     run_docker "create -v /srv/ --name pmm-data $repo:$tag /bin/true 1> /dev/null"
-    msg "Created PMM Data Volume: pmm-data"
+    msg "Created PMM Data Volume: pmm-data\n"
+  else 
+    msg "\n\t${YELLOW}Existing PMM Data volume found, renaming to pmm-data-$pmm_archive${NOFORMAT}\n"
+    run_docker "rename pmm-data pmm-data-$pmm_archive"
```

Comment:
shouldn't we ask user if he wants to use existing PMM Data volume or create new one? What if user will decide to use the same script to upgrade PMM?

Reply:
I didn't think about users doing upgrades with it (which I don't know that they should be...) so we kind of need to think of what the default behavior should be with zero interaction and then allow users to change our decisions with --interactive (which has our defaults set).  I mentioned in an earlier comment that I feel like best practice is to take a backup before we upgrade...there's no going back to a 2.26.0 once the databases in /srv are upgraded to 2.27.0 without a backup...thoughts?

Reply:
I think an upgrade should be the default option (without pmm-data container rename). If someone just ran the script, it shouldn't delete any data.